sklearn/cluster/hierarchical.py

```diff
@@ -541,13 +541,16 @@ def _average_linkage(*args, **kwargs):
 ###############################################################################
 # Functions for cutting  hierarchical clustering tree
 
-def _hc_cut(n_clusters, children, n_leaves):
+def _hc_cut(n_clusters, children, n_leaves,
+            distance_threshold=None, distances=None):
     """Function cutting the ward tree for a given number of clusters.
 
     Parameters
     ----------
     n_clusters : int or ndarray
         The number of clusters to form.
+        NOTE: You should set either ``n_clusters`` or ``distance_threshold``,
+        NOT both.
```

Comment:
This comment doesn't really make sense, if `distance_threshold` overrides `n_clusters`. I think it's fine to only have the comment under `distance_threshold` saying that it being set makes this ignored.